src/backoffice/documents/documents/tree/tree-item/document-tree-item.element.ts

```diff
@@ -0,0 +1,72 @@
+import { css, html, nothing } from 'lit';
+import { UUITextStyles } from '@umbraco-ui/uui-css/lib';
+import { customElement, property } from 'lit/decorators.js';
+import { UmbDocumentTreeItemContext } from './document-tree-item.context';
+import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
+import { DocumentTreeItemResponseModel } from '@umbraco-cms/backoffice/backend-api';
+
+@customElement('umb-document-tree-item')
+export class UmbDocumentTreeItemElement extends UmbLitElement {
+	static styles = [
+		UUITextStyles,
+		css`
+			#icon-container {
+				position: relative;
+			}
+
+			#icon {
+				vertical-align: middle;
+			}
+
+			#status-symbol {
+				width: 8px;
+				height: 8px;
+				background-color: blue;
+				display: block;
+				position: absolute;
+				bottom: 0;
+				right: 0;
+				border-radius: 100%;
+			}
+		`,
+	];
+
+	@property({ type: Object, attribute: false })
+	public item?: DocumentTreeItemResponseModel;
+
+	render() {
+		if (!this.item) return nothing;
+		new UmbDocumentTreeItemContext(this, this.item);
```

Comment:
As we have talked about on Slack this might be created too often?
I'm also affright of the readability, when a developer reads this it might not so clear when its replaced — which is also the potential problem, that its re-generateed too often.